physionet-django/user/models.py

```diff
@@ -1,13 +1,17 @@
+# from django.db.models import permalink
+from django.db import IntegrityError
```

Comment:
As mentioned above, this import should appear lower down in the list.

Reply:
fixed!